BackofficeBundle/EventSubscriber/ContentTypeSubscriber.php

```diff
@@ -62,6 +81,31 @@ public function preSetData(FormEvent $event)
     /**
      * @param FormEvent $event
      */
+    public function postSetData(FormEvent $event)
+    {
+        $form = $event->getForm();
+        $data = $event->getData();
+
+        $contentType = $this->contentTypeRepository->findOneByContentTypeIdInLastVersion($data->getContentType());
+        if (is_object($contentType)) {
```

Comment:
A more approppriate test could be checking $contentType is an instance of the good interface